README.md

```diff
@@ -13,7 +13,7 @@ Please find project details at [Wiki](https://github.com/Samsung/TizenRT/wiki) e
 > [Configuration Sets](#configuration-sets)
 
 ## Quick Start
-### Getting the toolchain
+### Getting the toolchain for Linux
```

Comment:
Except os type at end of zipped file name, no difference between linux and mac tab.
How about keep same tab just changing os type?
```
### Getting the toolchain

Get the build in binaries and libraries, [gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads/6-2017-q1-update)
Untar the gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2 and export the path like
    ```bash
tar xvjf gcc-arm-none-eabi-6-2017-q1-update-[OS Type].tar.bz2
export PATH=<Your Toolchain PATH>:$PATH
    ```
Be aware that recommanded toolchain is fully working on 64bits machine.
```

Reply:
For 3rd, that is not a part of TizenRT. That is a part of os usages so that we don't need to leave a guidance. That's why we don't leave ~/.bashrc on linux.

Reply:
Anyway I don't want REAME to become bigger. When we need to make tabs for all of each os, I think we had better split it from README.

Reply:
OK. I shall remove the ~/.bash_profile part.